docs/server-side/v2/api-reference/Rooms/create-via-api.mdx

```diff
@@ -31,8 +31,20 @@ curl --location --request POST 'https://api.100ms.live/v2/rooms' \
     "description": "This is a sample description for the room",
     "customer": "627cdddff2e4e30487862ad1",
     "recording_info": {
-        "enabled": false
-    },
+            "enabled": true,
+            "upload_info": {
+                "type": "<upload type: supported are s3, gs, oss>",
```

Comment:
remove this. we want to encourage users to use the template to define this and not the create room API

Reply:
we can check whether this fits better in policy APIs